build-settings/rollup.config.js

```diff
@@ -103,8 +113,9 @@ const createConfig = (
 
     const extensions = [".js", ".jsx", ".ts", ".tsx"];
 
+    const isSingleFile = inputFile != null;
     const config = {
-        output: createOutputConfig(name, format, file),
+        output: createOutputConfig(name, format, file, isSingleFile),
         input: makePackageBasedPath(name, inputFile || "./src/index.ts"),
```

Comment:
**suggestion (non-blocking):** we default to using `./src/index.ts` in the case that there is no `inputFile`.  It seems like there's still a single file that we're compiling in that case.  I think some of these variable names could be reworked (or maybe a new property on the config object that gets passed in) to help make it easier to understand what's going on.

Reply:
It's not about the input file, it's about the outputs.